src/plugins/data_view_field_editor/public/components/preview/field_preview_context.tsx

```diff
@@ -77,11 +89,15 @@ export const FieldPreviewProvider: FunctionComponent = ({ children }) => {
     fieldFormats,
   } = useFieldEditorContext();
 
+  const fieldPreview$ = useMemo(() => new Subject<FieldPreview[]>(), []);
```

Comment:
I think you wanted useRef here instead. useMemo is a performance optimization and not guaranteed to be stable